### PR TITLE
Provide the EZStream utility + libshout library which can stream medi…

### PR DESCRIPTION
**Pull request by digitalbox94**
_Motivation: Provide the EZStream utility + libshout library which can stream media data to an IceCast server.

### Checklist
- [x] Build rule `all-supported` completed successfully
- [] Package upgrade completed successfully
- [X] New installation of package completed successfully

### CODE REVIEW

**Comment by digitalbox94** [comment]
I have build this spk with all-supported and all spk were correctly generated without any error.
Not sure to get your point about the travis thing, but let me know if you need additions checkings about this PR.

**Comment by digitalbox94** [comment]
Is there anything else needed to hélp the review ?

**Comment by publicarray** [comment]
I don't think that there is anything else. You just need to wait for @ymartin59 to get to this PR

**Comment by digitalbox94** [comment]
Is Icecast will be updated to latest version 2.4.2 ?

File: cross/ezstream/PLIST [hunk 1]
```diff
@@ -0,0 +1,13 @@
+bin:bin/ezstream
+bin:bin/ezstream-file.sh
```
Comment (ymartin59) [inline cross/ezstream/PLIST]: shell scripts and xml files must be "rsc" for resources - not binary or shared object which will be stripped...

File: spk/ezstream/Makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+DESCRIPTION = EZStream is a command line source client for Icecast media streaming server 
+
+DISPLAY_NAME = ezstream
+
+HOMEPAGE   = http://www.icecast.org/
+LICENSE    = GPLv2
+
+STARTABLE = no
+
+INSTALLER_SCRIPT = src/installer.sh
```
Comment (ymartin59) [inline spk/ezstream/Makefile]: Please remove custom installer script, to take benefits of generic framework installer - as an example of link creation, please have a look at https://github.com/SynoCommunity/spksrc/tree/master/spk/tmux

**Comment by ymartin59** [comment]
@digitalbox94 Thanks for contribution. Please do not hesitate to submit a PR with icecast update. Probably time to remove the "beta" flag too...

**Comment by ymartin59** [comment]
@digitalbox94 Does it make sense to compile "ezstream" included into "icecast" package itself? (to reduce volume and avoid to build same dependencies multiple times)

**Comment by digitalbox94** [comment]
Changes have been done for EZStream and are proposed in a second commit. 
I hope it's OK since I'm not very familiar with github and PR :)

I agree putting EZStream and IceCast together is a good idea, I need first to verify if last IceCast version is compiling without any problem, add the ezstream part in icecast cross & spk, and check all is working fine by launching a testing radio on my NAS.

**Comment by digitalbox94** [comment]
Are the changes OK ? Or do I need to create a branch on my repo ?

**Comment by DigitalBox98** [comment]
This PR can be closed because another one is handling both EZStream and Icecast in the same package with latest Icecast version + UI wizard for server configuration

**Pull request closed without merging**